.github/workflows/metrics-collector.yml

```diff
@@ -0,0 +1,22 @@
+name: Github repo and issue stats collection
+on:
+  schedule:
+    - cron: "*/30 * * * *"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: "grafana/grafana-github-actions"
+          path: ./actions
+          ref: main
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+      - name: Run metrics collector
+        uses: ./actions/metrics-collector
```

Comment:
Q: shouldn't be `run: ./actions/metrics-collector`? just wondering :)

Reply:
no, uses points to action, and this an action, not a shell command